Out of date comment in overview (#659)
salsa-rs/salsa: book/src/overview.md

}
```

> Just as with an input, new values are created by invoking `Ast::new`, and the `new` for a tracked struct only requires a `&`-reference to the database:

Reviewer: ```suggestion
Just as with an input, new values are created by invoking `Ast::new`. The `new` function on a tracked struct only requires a `&`-reference to the database:
```
